FlowCrypt/Common/Services/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -18,23 +24,43 @@ protocol EncryptedStorageType {
 }
 
 final class EncryptedStorage: EncryptedStorageType {
+    enum Constants {
+        static let schemaVersion: UInt64 = 1
+        static let encryptedDBName = "encrypted.realm"
+    }
 
     let keychainService: KeyChainServiceType
-    private var canHaveAccessToStorage: Bool { accessCheck() }
-    private let accessCheck: () -> (Bool)
+    private let fileManager: FileManager
+
+    var isEncrypted: Bool {
```

Comment:
Should there ever be situation when `EncryptedStorage.isEncrypted` should return false?

Reply:
It's just an additional check to be sure and doesn't proceed farther if the implementation of KeychainService or EncryptedStorage will be changed in the future.

Reply:
I think we should not be creating checks for future code changes that have not yet occurred.  We may be building for future that never arrives, and creates code bloat/confusion. I'd rather see it removed, `EncryptedStorage` should always be encrypted.